packages/flutter/lib/src/material/button_style.dart

```diff
@@ -478,31 +478,31 @@ class ButtonStyle with Diagnosticable {
       return this;
     }
     return copyWith(
-      textStyle: textStyle ?? style.textStyle,
-      backgroundColor: backgroundColor ?? style.backgroundColor,
-      foregroundColor: foregroundColor ?? style.foregroundColor,
-      overlayColor: overlayColor ?? style.overlayColor,
-      shadowColor: shadowColor ?? style.shadowColor,
-      surfaceTintColor: surfaceTintColor ?? style.surfaceTintColor,
-      elevation: elevation ?? style.elevation,
-      padding: padding ?? style.padding,
-      minimumSize: minimumSize ?? style.minimumSize,
-      fixedSize: fixedSize ?? style.fixedSize,
-      maximumSize: maximumSize ?? style.maximumSize,
-      iconColor: iconColor ?? style.iconColor,
-      iconSize: iconSize ?? style.iconSize,
-      iconAlignment: iconAlignment ?? style.iconAlignment,
-      side: side ?? style.side,
-      shape: shape ?? style.shape,
-      mouseCursor: mouseCursor ?? style.mouseCursor,
-      visualDensity: visualDensity ?? style.visualDensity,
-      tapTargetSize: tapTargetSize ?? style.tapTargetSize,
-      animationDuration: animationDuration ?? style.animationDuration,
-      enableFeedback: enableFeedback ?? style.enableFeedback,
-      alignment: alignment ?? style.alignment,
-      splashFactory: splashFactory ?? style.splashFactory,
-      backgroundBuilder: backgroundBuilder ?? style.backgroundBuilder,
-      foregroundBuilder: foregroundBuilder ?? style.foregroundBuilder,
+      textStyle: style.textStyle,
```

Comment:
We might not want to update this `copyWith` method. It will cause massive breakages. I thought the change in the list_tile.dart is enough?